runtimes/runtimes/operational-telemetry/operational-telemetry-service.ts

```diff
@@ -0,0 +1,85 @@
+import { PeriodicExportingMetricReader } from '@opentelemetry/sdk-metrics'
+import { AWSMetricExporter } from './aws-metric-exporter'
+import { OperationalTelemetry } from './operational-telemetry'
+import opentelemetry, { Attributes } from '@opentelemetry/api'
+import { NodeSDK } from '@opentelemetry/sdk-node'
+import { Resource } from '@opentelemetry/resources'
+import { ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION } from '@opentelemetry/semantic-conventions/*'
+import { randomUUID } from 'crypto'
+
+export class OperationalTelemetryService implements OperationalTelemetry {
+    private customResource: Record<string, any> = {}
+    private static instance: OperationalTelemetryService
+    private SCOPE_NAME = 'language-server-runtimes'
+    private initialized = false
+
+    private constructor() {}
+
+    reportCounterMetric(counterName: string, value = 1, attributes?: Record<string, any>): void {
+        if (!this.initialized) {
+            // todo log diag error
+            return
+        }
+
+        const meter = opentelemetry.metrics.getMeter(this.SCOPE_NAME)
```

Comment:
Do we need to call `getMeter` in every call?

Reply:
Any recommendation for how to use it? It looks like something static that can be defined once to me.

Reply:
> It’s generally recommended to call getMeter in your app when you need it rather than exporting the meter instance to the rest of your app. This helps avoid trickier application load issues when other required dependencies are involved.
from https://opentelemetry.io/docs/languages/js/instrumentation/

Reply:
In the future, if we want to have different scopes, we would probably need to obtain the meter in the method anyway.